### PR TITLE
Prevent calling blocked numbers

### PR DESCRIPTION
**Pull request by traien**
Actually I think this feature is important because in latest versions of android many OEMs have added emergency contact button to the lock screen and there is no option to disable it, my kids keep pressing that button and calling 112 whenever found my device alone which put me in a lot of embarrassing situations, so I will be happy to see this feature on your amazing app however I think it is good to add fingerprint/pin code confirmation before calling a blocked number, thanks.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -64,6 +64,7 @@
     <string name="audio_route_wired_headset">Wired Headset</string>
     <string name="audio_route_wired_or_earpiece">Wired or Earpiece</string>
     <string name="choose_audio_route">Choose audio route</string>
+    <string name="calling_blocked_numbe">The calling number is blocked</string>
```
Comment (tibbi) [inline app/src/main/res/values/strings.xml:67]: fix the typo "calling_blocked_numbe" to "calling_blocked_number" and change it to "The number you are calling is blocked"

Comment (traien) [reply app/src/main/res/values/strings.xml]: Thank you, I have fixed them

**Comment by tibbi** [comment]
you also have to add new strings into every language file at the proper place, even if obviously in english

**Comment by traien** [comment]
> you also have to add new strings into every language file at the proper place, even if obviously in english

I have added them all on the `<!-- Dialer -->` section

**Comment by tibbi** [comment]
should be good, thanks

**Pull request merged**